web/src/components/TRIXO.js

```diff
@@ -0,0 +1,269 @@
+                value={data.kyc_threshold}
+                onChange={createChangeHandler("kyc_threshold")}
+              />
+            </Col>
+            <Col xs={3}>
+              <Form.Control
+                as="select" custom
+                value={data.kyc_threshold_currency}
+                onChange={createChangeHandler("kyc_threshold_currency")}
+              >
+                <Currencies />
+              </Form.Control>
+            </Col>
+          </Form.Row>
+          <Form.Text className="text-muted">
+            Threshold to conduct KYC before permitting the customer to send/receive virtual asset transfers
+          </Form.Text>
+        </Form.Group>
+        <Form.Group>
+          <Form.Label>Is your organization required to comply with the application of the Travel Rule standards in the jurisdiction(s) where it is licensed/approved/registered?</Form.Label>
+          <Form.Check
+            type="switch"
+            id="mustComplyTravelRule"
+            label="Must comply with the Travel Rule"
+            checked={data.must_comply_travel_rule}
+            onChange={createBoolChangeHandler("must_comply_travel_rule")}
+          />
+        </Form.Group>
+        <fieldset>
+          <Form.Label className="mb-0">Applicable Regulations</Form.Label>
+          <p className="text-muted mt-0">
+            <small>Please specify the applicable regulation(s) for Travel Rule standards compliance, e.g. "FATF Recommendation 16"</small>
+          </p>
+          {applicableRegulations}
+          <Form.Group>
+            <Button onClick={createArrayPushHandler("applicable_regulations", "")}>Add Regulation</Button>
+          </Form.Group>
+        </fieldset>
+        <Form.Group>
+          <Form.Label>What is the minimum threshold for Travel Rule compliance?</Form.Label>
+          <Form.Row>
+            <Col>
+              <Form.Control as={Col}
+                type="number"
+                value={data.compliance_threshold}
+                onChange={createChangeHandler("compliance_threshold")}
+              />
+            </Col>
+            <Col xs={3}>
+              <Form.Control
+                as="select" custom
+                value={data.compliance_threshold_currency}
+                onChange={createChangeHandler("compliance_threshold_currency")}
+              >
+                <Currencies />
+              </Form.Control>
+            </Col>
+          </Form.Row>
+          <Form.Text className="text-muted">
+            The minimum threshold above which your organization is required to collect/send Travel Rule information.
+          </Form.Text>
+        </Form.Group>
+      </fieldset>
+      <fieldset>
+        <legend className="sublegend">Data Protection Policies</legend>
+        <Form.Group>
+          <Form.Label>Is your organization required by law to safeguard PII?</Form.Label>
+          <Form.Check
+            type="switch"
+            id="mustSafegaurdPII"
```

Comment:
Safegaurd => Safeguard

Reply:
I will check to ensure that it's also spelled correctly in the protocol buffers!